static/css/footer.less

```diff
@@ -189,133 +121,118 @@
         position: relative;
         margin-bottom: unit(45px / @base-font-size-px, em);
 
+        .footer_top-button {
+            display: block;
+            width: 90%;
+            max-width: 350px;
+            margin: -2em auto 2em;
```

Comment:
Add code comment above this if this is for optical alignment of things... Eventually we'll want to standardize how you're doing optical hacks, so it would help to note them in the code. Ignore this comment if this's doing something different.